pkg/cmd/pulumi/up.go

```diff
@@ -505,6 +522,14 @@ func newUpCmd() *cobra.Command {
 		&yes, "yes", "y", false,
 		"Automatically approve and perform the update after previewing it")
 
+	if hasExperimentalCommands() {
+		cmd.PersistentFlags().StringVar(
+			&planFilePath, "constraints-file", "",
+			"Path to a constraints file to use for the update. The update will not "+
+				"perform operations that exceed its constraints (e.g. replacements instead of updates, or updates instead"+
+				"of sames).")
```

Comment:
```suggestion
			&planFilePath, "plan-file", "",
			"Path to a plan file to use for the update. The update will not "+
				"perform operations that exceed its plan (e.g. replacements instead of updates, or updates instead"+
				"of sames).")
```

Also, what does "sames" mean in this context?

Reply:
"sames" is that we don't expect any change